### PR TITLE
v. 2.0 feedback

### PR DESCRIPTION
**Pull request by jstorm31**
- Fixed copy link
- Fixed card dates
- Fixed icons/texts colours based on the design
- Improved responsivity

Resolves [#74059](https://redmine.ack.ee/issues/74059)

### CODE REVIEW

File: src/modules/filters/components/FilterBar/FilterBar.jsx [hunk 1]
```diff
@@ -58,7 +73,7 @@ const FilterBar = () => {
             >
                 <Row gutter={16} className={styles.row}>
                     {filterProps.map(select => (
-                        <Col key={nanoid()} span={8} lg={3}>
+                        <Col key={nanoid()} {...select.col}>
```
Comment (adamtretera) [inline src/modules/filters/components/FilterBar/FilterBar.jsx:76]: like this approach  👍

File: src/modules/ui/components/DataCard/DataCard.jsx [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
 import useFelaEnhanced from 'hooks/useFelaEnhanced';
-import { AndroidFilled, AppleFilled, CopyOutlined } from '@ant-design/icons';
+import { AndroidFilled, AppleFilled } from '@ant-design/icons';
 
 import DeleteButton from 'modules/reports/components/DeleteButton';
 import * as felaRules from './DataCard.styles';
-import Button from '../Button';
 import videoImg from 'assets/images/thumbnail_vid.png';
-import config from 'config/index';
+import { CopyButton } from 'modules/reports/components/CopyButton';
```
Comment (adamtretera) [inline src/modules/ui/components/DataCard/DataCard.jsx:11]: could be shortened, right ?

**Review by adamtretera — APPROVED** [review]
Good job 👏

File: src/modules/ui/components/DataCard/DataCard.jsx (hunk 2 above)
Comment (jstorm31) [reply src/modules/ui/components/DataCard/DataCard.jsx]: yep 👍

**Pull request merged**